pycqed/analysis_v2/tomography_dataprep.py

```diff
@@ -0,0 +1,250 @@
+
+import os
+import time
+from imp import reload
+from matplotlib import pyplot as plt
+import numpy as np
+from pycqed.analysis import measurement_analysis as MA 
+from pycqed.analysis import ramiro_analysis as RA
+from pycqed.analysis import fitting_models as fit_mods
+import lmfit
+import scipy as scipy
+try:
+	import qutip as qt
+except ImportError as e:
+	logging.warning('Could not import qutip, tomo code will not work')
+import itertools
+
+
+#Written By MALAY SINGH 
+#To prepare relevant inputs for tomography_V2
+
+
+
+class TomoPrep():
+	"""
+	class to prepare the necessary inputs for tomography_V2.
+	Does thresholding and coincidence counting on measured voltages
```

Comment:
So I figure this is in some way needed for the mav2 class for tomography. But how do they relate? 
Maybe add a comment here where that documentation (if it exists) can be found.